extensions/wikia/PhalanxII/templates/PhalanxSpecial_main.php

```diff
@@ -33,13 +33,13 @@
 						<!-- Format -->
 						<div class="clearfix">
 							<div class="left-spacer">&nbsp;</div>
-							<?= Xml::check( 'wpPhalanxFormatRegex', !empty( $data['regex'] ), array( 'id' => 'wpPhalanxFormatRegex' ) ) ?>
+							<input type="checkbox" id="wpPhalanxFormatRegex" value="1" <? if ( !empty( $data['regex'] ) ): ?>checked="checked" <? endif; ?>/>
```

Comment:
I'd also suggest moving the checkbox inside the label so there isn't a gap between the box and the text that isn't clickable.